cmd/query.go

```diff
@@ -205,10 +209,11 @@ func queryUnrelayedPackets(ctx *config.Context) *cobra.Command {
 			}
 			out, err := json.Marshal(sp)
 			if err != nil {
+				logger.Error("failed to marshal sequences", zap.Any("sp", sp), zap.Error(err))
```

Comment:
You need use `Errorw` to output a structured log, don't you?

Reply:
Use `Errorw`